client/src/components/PreviousTripsCard/PreviousTripsCard.test.js

```diff
@@ -1,16 +1,51 @@
 import React from 'react'
 import { shallow, configure } from 'enzyme'
 import PreviousTripsCard from './PreviousTripsCard.js'
-import Adapter from 'enzyme-adapter-react-16';
-configure({adapter: new Adapter()});
+import TripCard from 'components/TripCard/TripCard.js';
 
-// We will describe a block of tests
-describe('PreviousTripsCard component', () => {
-	// we will write one individual test
-  it('should render as expected', () => {
-    // Shallow rendering renders a component without rendering any of its children
-    const component = shallow(<PreviousTripsCard />)
-    // We create an assertion within the test that checks if our component renders our name prop
-  // expect(component.contains('Your name')).toBe(true)
+
+
+const setUp = (props, children) => {
+  const component = shallow(<PreviousTripsCard {...props}>{children}</PreviousTripsCard>);
+  return component;
+}
+describe("PreviousTripsCard component", () => {
+  describe('Has props', ()=>{
+    let wrapper;
+    beforeEach(()=>{
+      const props = {
```

Comment:
I'm noticing the describe is nest again, but I ran the code myself, and now I understand exactly what you're doing, this makes a lot of sense. Very cool and AMAZING job!